analytics_data_api/v1/urls.py

```diff
@@ -0,0 +1,5 @@
+from ..v0.urls import urlpatterns as v0_urlpatterns
+
+app_name = 'v1'
+
+urlpatterns = v0_urlpatterns.copy()
```

Comment:
this shell was much less code than I anticipated 😆

Reply:
I also think if we want to add new urlpatterns to v1, we can just do that by `+=` operators below this, right? That's python magic.